.github/workflows/build-container.yml

```diff
@@ -1,14 +1,11 @@
 name: "CI"
 on:
   push:
-    branches:
-      - devel
```

Comment:
Do we really need to run this in every branch? I'm not sure if Github applies quotas to Opensource projects, but in general Github Actions have the concept of quota... From a CI perspective, we only mind a few references (`devel` and perhaps tags/releases?).

This means that on every PR push 2 runs of the CI happen: one for the `push` event to the base branch, and another for the `pull_request` to the target `devel` branch...

I'd say that if we want to ensure code quality in `devel` (btw, maybe time to switch to `main`, which is the standard for the development branch) CI on `pull_request` events is enough.

Reply:
Thank you for review, Ernesto. It would be nice to see that functionality does not break in side branches, don't you think?  No worries about quotas: `GitHub Actions usage is free for standard GitHub-hosted runners in public repositories, and for self-hosted runners.` see [here](https://docs.github.com/en/billing/managing-billing-for-github-actions/about-billing-for-github-actions), since "public repositories" applies 😀.

Reply:
True, but it might become quite noisy. For every time that you submit or refresh a PR, you'll receive 2 Github Action status emails: 1 for the `push` event on the base branch and another for the `pull_request` one on the target branch... If you check examples, usually at least `pull_request` is only enabled in the main branch (or any branch admitting PRs), and `push` is only enabled in pushable branches.

Reply:
It is beneficial to detect issues, such as test failures, as early as possible. Personally, I find the concept of validating each commit appealing since it makes it easier to trace the root cause of problems later on. However, I agree that GitHub email notifications can be overwhelming, so I am currently considering how best to implement appropriate filtering in that regard. 😆